WordPress/build.gradle

```diff
@@ -331,7 +329,11 @@ dependencies {
     androidTestImplementation 'org.mockito:mockito-android:3.3.3'
     androidTestImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:$nhaarmanMockitoVersion"
     androidTestImplementation 'com.squareup.okhttp:mockwebserver:2.7.5'
-    testImplementation('org.bouncycastle:bcprov-jdk15on:1.64') { force = true }
+    testImplementation('org.bouncycastle:bcprov-jdk15on') {
+        version {
+            strictly '1.64'
```

Comment:
Side note: could be nice to add a comment explaining why we use strict versioning here (I'm assuming some bug we encountered in hotfix versions of the library or something, but having the rationale in a comment in that `build.gradle` would be nice)

Reply:
I don't know why we are using a strict version either. I'll have to track it down.

I'll try to remember to do this when I move the version numbers outside of the build files.